suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -63,9 +63,8 @@ function assertStaticSessionId(
     }
 }
 
-// discovery could have been deleted, or cancelled ín the meantime
-const canDiscoveryContinue = (discovery?: DiscoveryStatus) =>
-    discovery && isDiscoveryInProgress(discovery);
+// discovery could have been deleted, or cancelled in the meantime
+const canDiscoveryContinue = (discovery?: DiscoveryStatus) => isDiscoveryInProgress(discovery);
```

Comment:
isDiscoveryInProgress already takes optional discovery. Next step is to replace canDiscoveryContinue with isDiscoveryInProgress then..